### DISCUSSION TITLE
如何升级PCL2 的fabric loader, 但不会创建新版本.

### DISCUSSION THREAD

**Discussion by NotDanten** [Meloong-Git/PCL#4296] [提问] [answered]
如何升级PCL2 的fabric loader, 但不会创建新版本.
![image](https://github.com/user-attachments/assets/c8ab6540-f9c7-43f9-b801-d100d7ac242d)

**Comment by WForst-Breeze** [comment, marked as answer]
PCL 尚不支持该功能，可以去 #227 为其投票！

**Reply by NotDanten** [reply]
但有手动的方法吗

**Reply by hejiehao** [reply]
~~重装~~

**Reply by Telsolytor** [reply]
> 但有手动的方法吗

使用相同版本的Minecraft安装新的Fabric, 然后手动将旧版本地图、模组及其配置文件等转移到新版本中

**Reply by WForst-Breeze** [reply]
> 但有手动的方法吗

那恐怕就只能重新安装 Minecraft 并手动迁移存档和配置文件了

**Reply by Deep-Dark-Forest** [reply]
#227 不就是修改已安装的 Forge 和 OF 吗，也没说支持fabric啊

**Reply by MoYuan-CN** [reply]
> #227 不就是修改已安装的 Forge 和 OF 吗，也没说支持fabric啊

上游 #220，改标题后 #227 没有同步（

**Comment by hejiehao** [comment]
~~使用 HMCL~~

**Reply by allMagicNB** [reply]
https://github.com/HMCL-dev/HMCL/issues/3154#issuecomment-2194553157

**Comment by Silverteal** [comment]
1.安装一个新的目标版本
2.打开新目标版本的和待升级版本的json文件
3.将待升级版本json中除版本名外不同的部分全部替换为目标版本json文件的内容
4.删除待升级版本的主jar文件
5.“自动补全”待升级版本
6.完成，删除新的目标版本

以上步骤是我个人操作的，并不可靠，也不确保安全
